enginefs.js

```diff
@@ -201,6 +203,21 @@ router.all("/:infoHash/create", function(req, res) {
         res.end(JSON.stringify(getStatistics(engines[ih])));
     });
 });
+router.all("/create", function(req, res) {
+    var parsed
+    try {
+        var torrent = fs.readFileSync(req.body.from)
```

Comment:
@Ivshti `parse-torrent-file` requires to read the file, it does not allow just use the path. maybe something changed from previous versions. 
the documentation shows a use very similar to what i have done.

https://github.com/feross/parse-torrent-file